Automattic/jetpack
# Slideshow shortcode: Clean up jQuery usage and remove deprecated JS

Fixes https://github.com/Automattic/jetpack/issues/16715

## Proposed changes:

* This PR cleans up JS usage in the slideshow-shortcode.js file by removing deprecated functions. It also removes jQuery usage from much of the file, except where needed in order to integrate with the jQuery cycles plugin that is used. 
* To remove that, we'd need to use an alternative to that plugin (such as the Swiper library) and integrate that in.

### Other information:


## Jetpack product discussion

n/a

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

Self-hosted, WoA, Simple:
* To test, make sure Slideshow functionality via shortcode works as expected still. You can test this by adding a Shortcode block (make sure shortcodes are enabled at Jetpack > Settings > Writing). Add a gallery shortcode: `[gallery type="slideshow" ids="145,76,58" autostart="false"]` - replacing the IDs with several image ids from your site.
* The slideshow should also work with autostart enabled. Pressing play and pause should work as expected.
